AIM/sourcecode/FileHandler.py

```diff
@@ -175,20 +175,6 @@ def GenerateRunParameters(self, CallCommand):
             "\n\nRunning version " + self.script_version_print
         ])
 
-        
-        # self.backlog.append([
```

Comment:
I agree to this removal. Do we not actually print somewhere a reference to the published paper? If not that would be a good idea.

Reply:
Yes, we already do that.